Create ui-c8y-1021-81-0-tree-grid.md (#3451)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-81-0-tree-grid.md

---
date: ""

> title: Tree grid

Reviewer: ```suggestion
title: Data grid was extended to support display of hierarchical data (tree grid)
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4593
version: 1021.81.0
---

> Tree grid

Reviewer: The Data grid component has been updated to support hierarchical data structures. This enables a tree grid view where rows can be expanded to reveal nested child rows.

Key features include:
- Expand/collapse functionality for parent rows
- Indented row rendering for visual hierarchy
- Support for server-side and client-side data

This update allows for clearer representation of parent-child relationships in datasets such as device hierarchies and asset hierarchies.

> Tree grid

Reviewer: ```suggestion
The Data grid component has been updated to support hierarchical data structures. This enables a tree grid view where rows can be expanded to reveal nested child rows.

Key features include:

- Expand/collapse functionality for parent rows
- Indented row rendering for visual hierarchy
- Support for server-side and client-side data

This update allows for clearer representation of parent-child relationships in datasets such as device hierarchies and asset hierarchies.
```

---
date: ""

> title: Data grid was extended to support display of hierarchical data (tree grid)

Reviewer 2: ```suggestion
title: Data grid component now supports display of hierarchical data
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM

>     label: Device Management app

Reviewer 2: @bozhidardedov This should be Web SDK, shouldn´t it? Which would also mean to change the product area and move the file.

Reviewer: Sorry, my bad - wrong prefix in the merge commit. Should be in the right place now.

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4593
version: 1021.81.0
---
The Data grid component has been updated to support hierarchical data structures. This enables a tree grid view where rows can be expanded to reveal nested child rows.

Key features include:

- Expand/collapse functionality for parent rows
- Indented row rendering for visual hierarchy
- Support for server-side and client-side data

> This update allows for clearer representation of parent-child relationships in datasets such as device hierarchies and asset hierarchies.

Reviewer 2: ```suggestion
This update enables clearer representation of parent-child relationships in datasets, such as device hierarchies and asset hierarchies.
```
